src/transformer.js

```diff
@@ -44,7 +55,7 @@ const processSync = (source, filename, jestOptions) => {
   const preprocessor = require.resolve('./preprocess.js')
 
   const preprocessResult = execSync(
-        `node --unhandled-rejections=strict --abort-on-uncaught-exception "${preprocessor}"`,
+        `node --experimental-vm-modules --unhandled-rejections=strict --abort-on-uncaught-exception "${preprocessor}"`,
```

Comment:
is this supposed to be in this PR? it looks like what you were trying to add in #101 instead?

Reply:
I got it a little bit mixed up :) I will move some of my commits to #101 instead.

Reply:
Or is it okay, because both go hand in hand?

Reply:
this one seems better on #101 to me

Reply:
Okay, i am going to move it.